storage/aptosdb/src/lib.rs

```diff
@@ -2103,6 +2120,14 @@ impl DbWriter for AptosDB {
 
                 self.commit_ledger_info(last_version, new_root_hash, ledger_info_with_sigs)?;
 
+                // Record overall commit progress
+                let batch = SchemaBatch::new();
+                batch.put::<DbMetadataSchema>(
+                    &DbMetadataKey::OverallCommitProgress,
+                    &DbMetadataValue::Version(last_version),
+                )?;
+                self.ledger_db.metadata_db().write_schemas(batch)?;
```

Comment:
why? You've already made a separate method to special case the genesis LI?

Reply:
overall progress is separate from ledger progress since I may only want to commit ledger progress while the overall progress doesn't change

Reply:
I think the logic is "LI goes in last, together with the overall progress". I don't think it's necessary to change that -- especially not necessary in this PR since it's irrelevant with your introduction of the `commit_genesis_LI()` method?

@grao1991 ?

Reply:
deleted, since I no longer need to commit ledger progress at 0 separately now